diff --git a/node_modules/@emotion/styled/dist/declarations/types/base.d.ts b/node_modules/@emotion/styled/dist/declarations/types/base.d.ts
index c75842c..ef71ecb 100644
--- a/node_modules/@emotion/styled/dist/declarations/types/base.d.ts
+++ b/node_modules/@emotion/styled/dist/declarations/types/base.d.ts
@@ -5,6 +5,8 @@ import * as React from 'react'
 import { ComponentSelector, Interpolation } from '@emotion/serialize'
 import { PropsOf, Theme } from '@emotion/react'
 
+import type { JSX } from "react";
+
 export {
   ArrayInterpolation,
   CSSObject,
diff --git a/node_modules/@emotion/styled/dist/declarations/types/index.d.ts b/node_modules/@emotion/styled/dist/declarations/types/index.d.ts
index 883a79e..7be7fa0 100644
--- a/node_modules/@emotion/styled/dist/declarations/types/index.d.ts
+++ b/node_modules/@emotion/styled/dist/declarations/types/index.d.ts
@@ -4,6 +4,8 @@
 import { Theme } from '@emotion/react'
 import { CreateStyled as BaseCreateStyled, CreateStyledComponent } from './base'
 
+import type { JSX } from "react";
+
 export {
   ArrayInterpolation,
   ComponentSelector,
diff --git a/node_modules/@emotion/styled/types/base.d.ts b/node_modules/@emotion/styled/types/base.d.ts
index c75842c..ef71ecb 100644
--- a/node_modules/@emotion/styled/types/base.d.ts
+++ b/node_modules/@emotion/styled/types/base.d.ts
@@ -5,6 +5,8 @@ import * as React from 'react'
 import { ComponentSelector, Interpolation } from '@emotion/serialize'
 import { PropsOf, Theme } from '@emotion/react'
 
+import type { JSX } from "react";
+
 export {
   ArrayInterpolation,
   CSSObject,
diff --git a/node_modules/@emotion/styled/types/index.d.ts b/node_modules/@emotion/styled/types/index.d.ts
index 883a79e..7be7fa0 100644
--- a/node_modules/@emotion/styled/types/index.d.ts
+++ b/node_modules/@emotion/styled/types/index.d.ts
@@ -4,6 +4,8 @@
 import { Theme } from '@emotion/react'
 import { CreateStyled as BaseCreateStyled, CreateStyledComponent } from './base'
 
+import type { JSX } from "react";
+
 export {
   ArrayInterpolation,
   ComponentSelector,
