diff --git a/node_modules/@icons-pack/react-simple-icons/index.d.ts b/node_modules/@icons-pack/react-simple-icons/index.d.ts
index b3bfab2..899e67a 100644
--- a/node_modules/@icons-pack/react-simple-icons/index.d.ts
+++ b/node_modules/@icons-pack/react-simple-icons/index.d.ts
@@ -2732,3 +2732,4 @@ export { default as SiZorin, defaultColor as SiZorinHex } from './icons/SiZorin'
 export { default as SiZotero, defaultColor as SiZoteroHex } from './icons/SiZotero';
 export { default as SiZulip, defaultColor as SiZulipHex } from './icons/SiZulip';
 export { default as SiZyte, defaultColor as SiZyteHex } from './icons/SiZyte';
+export type * from "./types"
\ No newline at end of file
