diff --git a/node_modules/@vercel/style-guide/eslint/constants.js b/node_modules/@vercel/style-guide/eslint/constants.js
index a458b2f..5722c05 100644
--- a/node_modules/@vercel/style-guide/eslint/constants.js
+++ b/node_modules/@vercel/style-guide/eslint/constants.js
@@ -1,5 +1,5 @@
 module.exports = {
   ECMA_VERSION: 2021,
-  JAVASCRIPT_FILES: ['*.js?(x)', '*.mjs'],
+  JAVASCRIPT_FILES: ['*.js?(x)', '*.mjs', '*.cjs'],
   TYPESCRIPT_FILES: ['*.ts?(x)'],
 };
