diff --git a/node_modules/@vercel/style-guide/eslint/constants.js b/node_modules/@vercel/style-guide/eslint/constants.js
index a458b2f..df99ba2 100644
--- a/node_modules/@vercel/style-guide/eslint/constants.js
+++ b/node_modules/@vercel/style-guide/eslint/constants.js
@@ -1,5 +1,5 @@
 module.exports = {
-  ECMA_VERSION: 2021,
+  ECMA_VERSION: 2022,
   JAVASCRIPT_FILES: ['*.js?(x)', '*.mjs'],
-  TYPESCRIPT_FILES: ['*.ts?(x)'],
+  TYPESCRIPT_FILES: ['*.ts?(x)', '*.mts'],
 };
