diff --git a/node_modules/next/dist/lib/verify-typescript-setup.js b/node_modules/next/dist/lib/verify-typescript-setup.js
index 23998d8..da91cf4 100644
--- a/node_modules/next/dist/lib/verify-typescript-setup.js
+++ b/node_modules/next/dist/lib/verify-typescript-setup.js
@@ -72,11 +72,6 @@ const requiredPackages = [
         pkg: 'typescript',
         exportsRestrict: true
     },
-    {
-        file: '@types/react/index.d.ts',
-        pkg: '@types/react',
-        exportsRestrict: true
-    },
     {
         file: '@types/node/index.d.ts',
         pkg: '@types/node',
